pkl-core/src/main/java/org/pkl/core/packages/PackageResolvers.java

```diff
@@ -333,7 +333,7 @@ public byte[] getBytes(
         throw fileIsADirectory();
       }
       var entries = cachedEntries.get(packageUri);
-      // need to normalize here but not in `doListElments` nor `doHasElement` because
+      // need to normalize here but not in `doListElements` nor `doHasElement` because
```

Comment:
The same grammar rule that had me rewrite a `not`/`nor` above complained here, but I didn't have the energy to propose a rewrite, it would be something like:

      // need to normalize unlike `doListElements` and `doHasElement` which call `TreePathElement.getElement` which does normalization.